### PR TITLE
refacto function exports and definition

### PR DESCRIPTION
**Pull request by pixelfact**

### CODE REVIEW

File: components/features/auth0/guards/authentication-guard.tsx [hunk 1]
```diff
@@ -2,9 +2,9 @@ import { withAuthenticationRequired } from "@auth0/auth0-react";
 import { ReactElement } from "react";
 import ProjectsLoader from "src/App/Loaders/ProjectsLoader.tsx";
 
-const AuthenticationGuard = ({ children }: { children: ReactElement }) => {
+function AuthenticationGuard({ children }: { children: ReactElement }) {
   return children;
-};
+}
 
 export default withAuthenticationRequired(AuthenticationGuard, {
```
Comment (NeoxAzrot) [inline components/features/auth0/guards/authentication-guard.tsx:9]: Don't export default

File: components/features/auth0/guards/authentication-guard.tsx [hunk 2]
```diff
@@ -2,9 +2,9 @@ import { withAuthenticationRequired } from "@auth0/auth0-react";
 import { ReactElement } from "react";
 import ProjectsLoader from "src/App/Loaders/ProjectsLoader.tsx";
 
-const AuthenticationGuard = ({ children }: { children: ReactElement }) => {
+function AuthenticationGuard({ children }: { children: ReactElement }) {
```
Comment (NeoxAzrot) [inline components/features/auth0/guards/authentication-guard.tsx:5]: Need export ?

File: components/features/auth0/handlers/handle-login.ts [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import { RedirectLoginOptions } from "@auth0/auth0-react";
 
-export const handleLoginWithRedirect = async (
+export async function handleLoginWithRedirect(
   loginWithRedirectFunc: (options?: RedirectLoginOptions) => Promise<void>
-) => {
+) {
   await loginWithRedirectFunc();
   //   {
```
Comment (NeoxAzrot) [inline components/features/auth0/handlers/handle-login.ts:7]: Could uncomment ?

File: src/App/Layout/Header/ProfileButton/View.tsx [hunk 4]
```diff
@@ -12,7 +12,7 @@ import Button, { ButtonSize, ButtonType } from "src/components/Button";
 import { useSidePanel } from "src/hooks/useSidePanel";
 import { useStackContributorProfile, useStackPayoutInfo } from "src/App/Stacks/Stacks";
 import { useAuth0 } from "@auth0/auth0-react";
-import handleLogout from "components/features/auth0/handlers/handle-logout.ts";
+import { handleLogout } from "components/features/auth0/handlers/handle-logout.ts";
 import { useImpersonation } from "components/features/impersonation/use-impersonation.tsx";
 
 type Props = {
```
Comment (NeoxAzrot) [inline src/App/Layout/Header/ProfileButton/View.tsx:18]: Interface instead of type

File: src/App/Layout/Header/ProfileButton/ViewMobile.tsx [hunk 5]
```diff
@@ -19,7 +19,7 @@ import { Fields } from "src/_pages/Rewards/UserRewardTable/Headers";
 import MeApi from "src/api/me";
 import { useStackContributorProfile, useStackPayoutInfo } from "src/App/Stacks/Stacks";
 import { useAuth0 } from "@auth0/auth0-react";
-import handleLogout from "components/features/auth0/handlers/handle-logout.ts";
+import { handleLogout } from "components/features/auth0/handlers/handle-logout.ts";
 import { useImpersonation } from "components/features/impersonation/use-impersonation.tsx";
 
 type Props = {
```
Comment (NeoxAzrot) [inline src/App/Layout/Header/ProfileButton/ViewMobile.tsx:25]: Interface instead of type

**Review by haydencleary — APPROVED** [review]

File: components/features/auth0/guards/authentication-guard.tsx (hunk 2 above)
Comment (pixelfact) [reply components/features/auth0/guards/authentication-guard.tsx:5]: exported with the HOC bellow

File: components/features/auth0/handlers/handle-login.ts (hunk 3 above)
Comment (pixelfact) [reply components/features/auth0/handlers/handle-login.ts:7]: still need to find a way to handle the redirecting to the state url

File: src/App/Layout/Header/ProfileButton/View.tsx (hunk 4 above)
Comment (pixelfact) [reply src/App/Layout/Header/ProfileButton/View.tsx:18]: old stuff, will be refacor when migrating to NextJS following the new conventions

File: src/App/Layout/Header/ProfileButton/ViewMobile.tsx (hunk 5 above)
Comment (pixelfact) [reply src/App/Layout/Header/ProfileButton/ViewMobile.tsx:25]: same

**Review by pixelfact — COMMENTED** [review]
done

File: components/features/auth0/guards/authentication-guard.tsx (hunk 1 above)
Comment (pixelfact) [reply components/features/auth0/guards/authentication-guard.tsx:9]: the idea seams not happy with the named export for the HOCs !

**Pull request merged**